jboss-fuse-sap-tool-suite/targetplatform/pom.xml

```diff
@@ -10,7 +10,7 @@
 		<version>10.0.0-SNAPSHOT</version>
 	</parent>
 
-	<artifactId>org.fusesource.ide.sap.targetplatform</artifactId>
+	<artifactId>sap-tooling-multiple</artifactId>

```

Comment:
I don't think that it is used anywhere else, I just tried to have coherent name with what has been changed in Fuse Tooling too.